docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  ms: &ms
+    build:
+      context: .
```

Comment:
@defektive I think we also need a `msfvenom` image or another entrypoint in the main docker file so we can use msfvenom in docker too